src/modules/extendedCallWindow/i18n/en_AU.root.json

```diff
@@ -0,0 +1,100 @@
+{
+    "name": "Extended alarm window",
+    "settings": {
+        "alarmIcons": {
+            "description": "Leave icons in the \"Alarm\" Show button, based on your selection. You can choose between <a href=\"https://fontawesome.com/icons?d=gallery&m=free\" target=\"_blank\">all free icons from Font-Awesome</a>. It is enough to specify the name of the icon. A live preview will then show you the icon.",
+            "icon": "Icon",
+            "preview": "Preview",
+            "style": "icon type",
+            "title": "alarm icons",
+            "vehicles": "vehicle types"
+        },
+        "alarmTime": {
+            "description": "Shows next to the Alarm button how long the vehicle with the longest journey time takes.",
+            "title": "Longest journey time"
+        },
+        "arrClickHighlight": {
+            "description": "Draws a frame around already clicked AARs.",
```

Comment:
And here use "outline" or "border" (see above) instead of "frame".
Plus the abbr is "ARR" 😉